src/popupmenu.c

```diff
@@ -781,13 +790,14 @@ pum_redraw(void)
 			if (pum_rl)
 			{
 			    screen_puts_len((char_u *)"  ", 2, row, col - 1,
-									attr);
+								    orig_attr);
 			    col -= 2;
 			}
 			else
 #endif
 			{
-			    screen_puts_len((char_u *)"  ", 2, row, col, attr);
+			    screen_puts_len((char_u *)"  ", 2, row, col,
+								    orig_attr);
```

Comment:
This change seems wrong. `attr` should be used here, not `orig_attr`

Reply:
because the space padding no need combine with new attr like `strikethrough` will looks wired

Reply:
It's not space padding.

Reply:
I don't think this is space padding. A TAB is a part of the text.

Reply:
yes you are right. i mixed up it